pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/executor/LeafSchedulerService.java

```diff
@@ -31,17 +34,20 @@
 import org.slf4j.LoggerFactory;
 
 
-public class LeafSchedulerService {
+public class LeafSchedulerService implements SchedulerService {
   private static final Logger LOGGER = LoggerFactory.getLogger(LeafSchedulerService.class);
 
   private final ExecutorService _executorService;
   private final ConcurrentHashMap<OpChainId, Future<?>> _submittedOpChainMap;
```

Comment:
dont see this getting cleaned up

Reply:
the goal for this POC is not cleaning this up, the solution is similar to mailbox service, will clean up using a expiry cache or clean up upon exit the opChain

Reply:
please add a todo for the closeOpChain then